srcpkgs/droidcam/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'droidcam'
+pkgname=droidcam
+version=1.7.1
+revision=1
+makedepends="pkg-config libjpeg-turbo-devel ffmpeg-devel alsa-lib-devel speex-devel libusbmuxd-devel libplist-devel gtk+3-devel"
+depends="v4l2loopback"
+short_desc="Tool for using your android device as a wireless/usb webcam"
+maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/dev47apps/droidcam"
+distfiles="https://github.com/dev47apps/${pkgname}/archive/v${version}.tar.gz"
+checksum="1f925c093df165afcd39e76b12a3db0f19fabbf15a11e1c7b55a97bba331f113"
+nopie=true
+
+do_build() {
+	# All JPEG* parameters are needed to use shared version of libturbojpeg instead of
+	# static one.
+	#
+	# Also libusbmuxd requires an override while linking.
+	make JPEG_DIR="" JPEG_INCLUDE="" JPEG_LIB="" JPEG=$(pkg-config --libs --cflags libturbojpeg) USBMUXD=-lusbmuxd-2.0
```

Comment:
If possible, it would be nicer to use `make_build_args` instead of `do_build`. You'd need to append to `make_build_args` in `pre_build`, though, since `pkg-config` won't be available before that.

Btw, `JPEG_DIR=""` and `JPEG_DIR=` do the same thing :)

Reply:
Using the gnu-makefile build style fails building with this:
```
g++ cc src/droidcam-cli.c src/connection.c src/settings.c src/decoder.c src/decoder_snd.c src/decoder_v4l2.c src/av.c src/usb.c -lturbojpeg `pkg-config --libs --cflags libswscale libavutil` -lspeex -lasound -lpthread -lm -lusbmuxd-2.0 -o droidcam-cli
g++: error: cc: No such file or directory
make: *** [Makefile:42: droidcam-cli] Error 1
=> ERROR: droidcam-1.7.1_1: do_build: '${make_cmd} CC="$CC" CXX="$CXX" LD="$LD" AR="$AR" RANLIB="$RANLIB" CPP="$CPP" AS="$AS" OBJCOPY="$OBJCOPY" OBJDUMP="$OBJDUMP" CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" ${makejobs} ${make_build_args} ${make_build_target}' exited with 2
=> ERROR:   in do_build() at common/build-style/gnu-makefile.sh:9
```

I have no idea where its getting "g++ cc" from though, cc isnt mentioned in the upstream makefile

Reply:
Apply https://github.com/dev47apps/droidcam/pull/146 as a patch, it should just work for us then.